tdrs-frontend/.env.development

```diff
@@ -1,3 +1,8 @@
+###
+# Development environment-specific settings
+# This file is loaded when running `npm start`
```

Comment:
Note, none of these values are different than the defaults. We could leave this file empty and it would just defer to the default `.env` file. 

Also, this file only gets loaded when running npm or yarn start directly (ie. no docker) - not to be confused with the `development` environment in cloud.gov